tests/statusHelper.ts

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2018 Lindorff Oy
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Status } from "../lib/interfaces";
+export class StatusHelper {
```

Comment:
This could be declared as `export default class StatusHelper` instead, especially since this is the only thing being exported.

This is way the importing class can use `import StatusHelper from 'statusHelper'`, making the code a bit cleaner and idiomatic.

Reply:
Yes, the Jira class could/should have the class as a default export as well. It can even be argued that the class wouldn't be needed, but the functions should be imported individually by the scripts... But I guess that's a discussion for another time and place